src/Http/Requests/Contact/StoreContactRequest.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Http\Requests\Contact;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class StoreContactRequest extends FormRequest
+{
+    public function rules(): array
+    {
+        return [
+            "name" => "required",
+            "email" => "required|email",
+            "subject" => "required",
+            "message" => "required",
```

Comment:
If we are  using in `FormRequest` classes array of strings, in that class we should do this also.